apps/site/util/getNodeDownloadSnippet.ts

```diff
@@ -75,6 +75,7 @@ export const getNodeDownloadSnippet = (
 
       # ${t('layouts.download.codeBox.downloadAndInstallNodejs')}
       brew install node@${release.major}
+      ${['Current', 'Pending'].includes(release.status) ? 'brew install corepack' : ''}
```

Comment:
hmm... maybe better to avoid the extra empty line for the older releases:

```suggestion
      brew install node@${release.major}${['Current', 'Pending'].includes(release.status) ? '\nbrew install corepack' : ''}
```